tests/functional/idempotency/test_idempotency.py

```diff
@@ -1363,7 +1361,7 @@ def two(data):
 
     assert one(data=mock_event) == "one"
     assert two(data=mock_event) == "two"
-    assert len(persistence_store.client.method_calls) == 4
+    assert len(persistence_store.client.method_calls) == 0
```

Comment:
The behavior when the key is `None` changed, so I had to change this test. I would like your opinion here.